components/simplified-grid/simplified-grid.css

```diff
@@ -0,0 +1,28 @@
+@import url("/docs/main.css");
+
+.grid_container {
+    display: grid;
+    background-color: var(--light-grey-color);
+    width: fit-content;
+    height: fit-content;
+}
+
+.grid_item {
+    padding: 0.5em;
+} 
+
+.grid_img {
+    height: 100%;
+}
+
+.grid_two_columns {
+    grid-template-columns: 1fr 1fr;
+    gap: 10px;
```

Comment:
Looks good, btw you can use  relative units instead of  px  here

Reply:
using**

Reply:
Yea, I'll will use that. Thanks!